### ISSUE TITLE
weird behavior of ResNet101 Quantization

### ISSUE THREAD

**Issue by zacurr** [pytorch/vision#2648]
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

Following the tutorial (https://pytorch.org/tutorials/advanced/static_quantization_tutorial.html), 
I tried to quantize Res18, 34, 50, 101 and 152 with both  fbgemm and qnnpack option.
By loading pre-trained float model, call quantize_model in quantization/utils.py
but I calbirated model with imagenet dataset (not a random tensor as in utils.py)

Evaluation results with imagenet1k set (introduced in tutorial)
Top-1 Accuracy

Res-18 : 68.4
Res-34 : 74
Res-50 : 77
Res-101 : 33.1 (calib with test set) / 52.9 (calib with train set)
Res-152 : 75.5

Only performance of Res-101 degraded much.

Do you have any idea about this?

**Comment by zacurr** [comment]
This is related to the following issue that i opened
https://github.com/pytorch/vision/issues/2849

If i change activation of QConfig from default min/max to histogram observer, 
performance is fine. 
Res-101: 75.6
